### DISCUSSION TITLE
recover 'project' file uploaded before

### DISCUSSION THREAD

**Discussion by Micaleiner** [community/community#38978] [New to GitHub]
I am trying to get a copy of the Mobirise 'project' file that I uploaded in order to publish changes to my website - can this be done by doing a download from the repository or any other way?

I would greatly appreciate your help

![Screenshot (1)](https://user-images.githubusercontent.com/61084742/201518240-a01fd03f-125d-493b-b60a-d41e0c188d32.png)

**Comment by Rajahussa** [comment]
Dont show to everyone

**Comment by airtower-luna** [comment]
In general the best way is to use Git on your local computer to *clone* the repository, make the changes you want, *commit* them, and then *push* them back to the GitHub repository, which should update your site.

If those terms all sound unfamiliar check the [Git Book](https://git-scm.com/book/en/v2) site for an introduction to Git. If you're in a hurry, the first 2 or 3 chapters are enough to start with. :wink:

**Comment by Micaleiner** [comment]
Hi Rajahussa

THANKS SO MUCH - I downloaded (zipped)  the repository and found the most
recent 'project' file amongst other data.  uploaded it to my website
builder and it was back to what it should be!
